mono/metadata/coree.c

```diff
@@ -630,7 +630,7 @@ STDAPI MonoFixupCorEE(HMODULE ModuleHandle)
 #if defined(_M_IA64)
 			ProcRva = (DWORD)((DWORD_PTR)PLabel - (DWORD_PTR)DosHeader);
 			*(PLabel)++ = *ExportFixup->ProcAddress.PLabel;
-#elif defined(_M_AMD64)
+#elif defined(_M_AMD64) || defined(_M_ARM64)
```

Comment:
I would suggest not defining `HAVE_API_SUPPORT_WIN32_COREE` for ARM64.

See where it's defined here: https://github.com/Unity-Technologies/mono/blob/ffd005d73712c4aa5fd2d5f36105c79855754e40/mono/utils/w32subset.h#L81